lib/libraries/c3/extensions/tooltip-contents.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (d, defaultTitleFormat, defaultValueFormat, color) {
+  var $$ = this, config = $$.config,
+      titleFormat = config.tooltip_format_title || defaultTitleFormat,
```

Comment:
This fugly little function is a literal copy-paste of c3's internal tooltip code, which can be customized and re-injected as a config option. It's admittedly super gross, but just haven't spent time untangling it yet.